frontend/src/lib/taxonomy.tsx

```diff
@@ -1212,6 +1212,10 @@ export const CORE_FILTER_DEFINITIONS_BY_GROUP = {
             description: 'posthog-js adds these to the page leave event, they are used in web analytics calculations',
             label: 'Previous pageview duration',
         },
+        $surveys_activated: {
```

Comment:
@Phanatic I took a guess at a description here, feel free to correct :)

(at the risk of teaching you to suck eggs... adding event properties here means when we display them we add a logo at the front and show the description in a pop over so folk know they're legit PH properties)

Reply:
I have no idea how to eat eggs, I just end up scrambling them :)
Good callout on the event properties! Sounds good to me.